src/styles/components/_name-generator.scss

```diff
@@ -39,11 +39,24 @@
   &__add-person {
     @include svg-button('/images/add-a-person.svg');
     position: absolute;
-    bottom: 20px;
-    right: 0;
-    width: 150px;
+    bottom: 35px;
```

Comment:
Do some of these numbers make sense to extract as a variable? e.g. 35px seems to be used multiple times; if those usages are connected a variable would make sense.
As a side note, are we linting the scss code?